src/main/archetype/ui.frontend/src/main/webpack/components/content/accordion/scss/accordion.scss

```diff
@@ -0,0 +1,39 @@
+/* BEM Description
```

Comment:
@pauli, I would simplify these by:
• Removing the BEM description (it doesn't really provide much more info than the class names and is already available in the component READMEs, so we don't really want to have to maintain it here too).
• Listing each selector in full (rather than using the `&`) - more readable.
• Putting one selector per line so the full file is readable straight off.

So for accordion, simply:
```
.cmp-accordion {}
.cmp-accordion__item {}
.cmp-accordion__header {}
.cmp-accordion__button {}
.cmp-accordion__button--expanded {}
.cmp-accordion__title {}
.cmp-accordion__icon {}
.cmp-accordion__panel {}
.cmp-accordion__panel--expanded {}
.cmp-accordion__panel--hidden {}
```

wdyt?